apps/content_script/appbar.ts

```diff
@@ -42,10 +43,13 @@ function showBlockedByBannedWords(): void {
     rows: 10,
   });
 
-  const bannedWordsDiv = document.getElementById(
-    "banned-words-div",
-  ) as HTMLDivElement;
-  bannedWordsDiv.appendChild(textarea);
+  const bannedWordsDiv = document.getElementById("banned-words-div");
+  if (bannedWordsDiv === null) {
+    Logger.debug("topstuff: no banned words div found");
+    return;
+  } else {
+    bannedWordsDiv.appendChild(textarea);
+  }
```

Comment:
@vibh1103 

The element with id=banned-words-div is created before this function is called.
https://github.com/ikemo3/gossip-site-blocker/blob/b0cac90e288c2ca84e9f401b5891fbe403c59bf6/apps/content_script/appbar.ts#L114-L116

Therefore, not finding this element is clearly a bug.
Instead of logging for such obvious bugs, an error should be thrown.
Options and popups should look like this.

```suggestion
  if (bannedWordsDiv === null) {
    throw new Error("topstuff: no banned words div found");
  }

  bannedWordsDiv.appendChild(textarea);
```